### ISSUE TITLE
Water Purifier Container

### ISSUE THREAD

**Issue by DanStaal** [DanStaal/DStaal_UKS_KPBS_Addons#21]
Config update needed.

Habitation/Recycling Info:

| Part                     | PartName                                   | Mass    | Volume   | Seats | Months | Multiplier | EC/s                  | Recycler % | Spreadsheet Approved? |
|:-------------------------|:-------------------------------------------|---------|----------|-------|--------|------------|-----------------------|------------|-----------------------|
| Water Purifier Container | KKAOSS\_LS\_container\_waterpurifier       | 0.7     | 1.2      | 0     | n/a    | n/a        | n/a                   | n/a        | no                    |

**Comment by Merkov** [comment]
Copied from the USI-LS Information Issue: 

"The Water Purifier has the same size and mass [as the recycler]. If we made it an 80% (0.006 Water) or 85% (0.007 Water) purifier for one kerbal (The EC/s cost goes up a lot in that small range: 0.5 EC/s for 80%, 1.75 EC/s for 85%) then it would be roughly balanced (80% is actually smaller and lighter [than the spreadsheet's guidelines], while 85% is a bit bigger and lighter than balanced, but I'm worried that 85% might be too high...). My thinking is that this could be a slightly later tech tier, only affects 1 kerbal, and uses water, but it can be used to boost a base's max recycling efficiency a bit before you get the really powerful purifier on the central hub." 

Do either of those sound good?

**Comment by DanStaal** [comment]
I'm also on 85% being a bit high, in comparison to the Central Hub.  On the other hand 80% is the high end for recyclers.  Can we split the difference?  Set it at say 82%?  Probably 0.007 water (shouldn't be and issue with KPBS's dedicated water drill), though I'm not sure how the EC usage should balance in.

**Comment by Merkov** [comment]
Throwing those numbers in, an 82% recycler for 1 kerbal would require 0.0064 Water/s and 1.0 EC/s. That sounds good to me.

**Comment by Merkov** [comment]
I have added a draft of a config for the water purifier. I've actually modified Nils' quite a bit. Since the part was originally only meant for TAC LS, it didn't put the TAC-specific module behind a :NEEDS... thing. The whole part needed TAC LS, but I had to add USI-LS to that. I think I have successfully moved the TAC module into a proper TAC only block, and put the USI LS purifier converter into a USI LS only block. 

I also made it so that you get this part with Short Term Habitation. By default, it was going to be available in the recycling node, but that feels a bit early in the CTT. I think I'm okay with the stock tech tree placement of Advanced Electrics.
